example/lambdas/ftpPopulateTestLambda/webpack.config.js

```diff
@@ -27,7 +27,7 @@ module.exports = {
     }
   },
   externals: [
-    'aws-sdk',
+    '@aws-sdk',
```

Comment:
I should probably know this but what are we saying here? What does this change do?

Reply:
Punting this conversation to https://github.com/nasa/cumulus/pull/3618#discussion_r1561655895